applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/residual_based_relaxation_scheme.hpp

```diff
@@ -295,7 +295,7 @@ class ResidualBasedRelaxationScheme : public Scheme<TSparseSpace, TDenseSpace>
     ) override
     {
         KRATOS_TRY
-        int k = OpenMPUtils::ThisThread();
+        int k = ParallelUtilities::GetThreadId();
         //Initializing the non linear iteration for the current element
         //KRATOS_WATCH( LHS_Contribution )
```

Comment:
watch?

Reply:
this one is not portable...

Reply:
This is outdated, see message 

![image](https://github.com/KratosMultiphysics/Kratos/assets/19991680/e7133429-fd8d-41f0-af7f-d7ce4406581a)